Sources/Logging/Strings/PaywallsStrings.swift

```diff
@@ -35,6 +45,27 @@ extension PaywallsStrings: LogMessage {
 
         case let .error_prefetching_image(url, error):
             return "Error pre-fetching paywall image '\(url)': \((error as NSError).description)"
+
+        // MARK: - Events
+
+        case .event_manager_not_initialized_not_available:
+            return "Won't initialize PaywallEventsManager: not available on current device."
+
+        case let .event_manager_failed_to_initialize(error):
+            return "PaywallEventsManager won't be initialized, event store failed to create " +
+            "with error: \((error as NSError).localizedDescription)"
+
+        case .event_flush_already_in_progress:
+            return "Paywall event flushing already in progress. Skipping."
+
+        case .event_flush_with_empty_store:
+            return "Paywall event flushing requested with empty store."
+
+        case let .event_flush_starting(count):
+            return "Paywall event flush: posting \(count) events."
+
+        case let .event_flush_failed(error):
+            return "Paywall event flushing failed, will retry. Error: \(error.localizedDescription)"
```

Comment:
I'm a big fan of your event/error messages 😀  I really need to want to do this more myself 😛

Reply:
As an SDK with no UI (well, you know what I mean 🤣) logs are so important. I love having `verbose` logs now that can have a lot of this information and most people won't see, but we can still look into when looking into bugs or when running tests, etc.